pom.xml

```diff
@@ -398,6 +426,32 @@
                 <artifactId>hamcrest</artifactId>
                 <version>${hamcrest.version}</version>
             </dependency>
+
+            <dependency>
```

Comment:
Super nIt: why not pair up the Javax and Jakarta dependencies next to one another like you did with `org.glassfish`?

Reply:
No reason. It was just convenient to track what was missing while working on it. I paired the dependencies up now.